xla/service/gpu/gemm_rewriter.cc

```diff
@@ -1516,7 +1617,7 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
     const Shape &output_shape = instr.shape();
 
     TF_ASSIGN_OR_RETURN(bool types_are_supported_by_cublas_lt,
-                        TypesAreSupportedByCublasLt(instr));
+                        AreTypesSupportedByCublasLt(instr));
```

Comment:
nit: any reason to change the function name from `TypesAre` to `AreTypes`? I think the original reads better, especially now with the inconsistent naming of the bool.

Reply:
Not really. Just renamed to `AreTypes*`